python/tk_framework_adobe/rpc/communicator.py

```diff
@@ -565,9 +567,9 @@ def _handle_response(self, response, *args):
         self.log_network_debug("Response UID is %s" % uid)
 
         try:
-            self._RESULTS[uid] = self._ensure_utf8(json.loads(result["result"]))
+            self._RESULTS[uid] = json.loads(six.ensure_str(result["result"]))
```

Comment:
This to me feels like it was a bug previously. I think they meant to ensure utf8 on the string before json loading it, but since the method it called checked if the passed "string" was a unicode instance, it wouldn't convert the json to unicode anyway.